module/VuFind/src/VuFind/Db/Entity/Feedback.php

```diff
@@ -138,59 +176,123 @@ class Feedback implements EntityInterface
      *
      * @ORM\ManyToOne(targetEntity="VuFind\Db\Entity\User")
      * @ORM\JoinColumns({
-     *   @ORM\JoinColumn(name="updated_by", referencedColumnName="id")
+     * @ORM\JoinColumn(name="updated_by",
+     *              referencedColumnName="id")
      * })
      */
     protected $updatedBy;
 
+    /**
+     * Message setter
+     *
+     * @param string $message Message
+     *
+     * @return Feedback
+     */
     public function setMessage(string $message): Feedback
     {
         $this->message = $message;
         return $this;
     }
 
+    /**
+     * Form data setter.
+     *
+     * @param string $data Form data
+     *
+     * @return Feedback
+     */
     public function setFormData(string $data): Feedback
     {
         $this->formData = $data;
         return $this;
     }
 
+    /**
+     * Form name setter.
+     *
+     * @param string $name Form name
+     *
+     * @return Feedback
+     */
     public function setFormName(string $name): Feedback
     {
         $this->formName = $name;
         return $this;
     }
 
+    /**
+     * Created setter.
+     *
+     * @param datetime $dateTime Created date
```

Comment:
Here and in the setter below, `DateTime` should be capitalized to match the actual type name.